src/Networking/FakeCANBoard.cpp

```diff
@@ -73,5 +87,33 @@ int main() {
       AssemblePIDTargetSetPacket(&p, motor_group, (uint8_t) serial, angle_target);
       sendCANPacket(p);
     }
+
+    if (test_type == TEST_SCIENCE_TELEMETRY) {
+        // Serial 0 seems to work. Nothing else (up to 17, the largest I tried)
+        serial = prompt("Enter serial");
+        uint8_t sensor_code = 42; // Nonsense for now, just testing CAN connection
+        AssembleScienceSensorPullPacket(&p, science_group, (uint8_t) serial, sensor_code);
+        sendCANPacket(p);
+    }
+
+    if (test_type == TEST_SCIENCE_MOTORS) {
+        // CAREFUL, there are no safety checks / limit switches
```

Comment:
What should we do about the fact that there are no limit switches? We might just have to leave it up to the operator to not go over, but not having some kind of failsafe doesn't sound great :grimacing: